### PR TITLE
test(case): optimize test case

### PR DESCRIPTION
**Pull request by forfreeday**
**What does this PR do?**
add several test cases to cover more code verification scenarios

**Why are these changes required?**
to improve test case coverage

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                "--output-directory", dbPath,
+            },
+            Constant.TEST_CONF
+    );
+  }
+
+  @Before
+  public void init() {
+    accountAssetStore.put(ASSET_KEY, Longs.toByteArray(200L));
+
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java:59]: No need to add blank line

Comment (forfreeday) [reply framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java:59]: This blank line does not affect the logic.

File: framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                    .setDescription(ByteString.copyFrom(ByteArray.fromString(DESCRIPTION)))
+                    .setUrl(ByteString.copyFrom(ByteArray.fromString(URL)))
+                    .build();
+    AssetIssueCapsule assetIssueCapsule = new AssetIssueCapsule(assetIssueContract);
+    chainBaseManager.getAssetIssueV2Store()
+            .put(assetIssueCapsule.createDbV2Key(), assetIssueCapsule);
+    try {
+      ownerCapsule.addAssetV2(ByteArray.fromString(String.valueOf(id)), TOTAL_SUPPLY);
+    } catch (Exception e) {
+      e.printStackTrace();
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java:91]: Should this be thrown?

File: framework/src/test/java/org/tron/core/db/ContractStoreTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  @Before
+  public void init() {
+    SmartContractOuterClass.SmartContract.Builder contract =
+            createContract(CONTRACT_ADDRESS, SMART_CONTRACT_NAME);
+    contractStore.put(
+            ByteArray.fromHexString(CONTRACT_ADDRESS),
+            new ContractCapsule(contract.build()));
+  }
+
+  private SmartContractOuterClass.SmartContract.Builder createContract(
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/ContractStoreTest.java:46]: Why don't return a contractCapsule?

File: framework/src/test/java/org/tron/core/db/AssetIssueV2StoreTest.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                            .setName(ByteString.copyFrom(firstTokenId.getBytes()))
+                            .build());
+    assetIssueCapsule.setId(String.valueOf(1L));
+    assetIssueV2Store
+            .put(assetIssueCapsule.createDbV2Key(), assetIssueCapsule);
+  }
+
+  @Test
+  public void put() {
+    String firstTokenId = "efg";
```
[the remark this answers is not in the document]
Comment (forfreeday) [reply framework/src/test/java/org/tron/core/db/AssetIssueV2StoreTest.java:48]: This has nothing to do with code logic

File: framework/src/test/java/org/tron/core/db/AssetIssueStoreTest.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  @Before
+  public void init() {
+    long id = dbManager.getDynamicPropertiesStore().getTokenIdNum() + 1;
+    AssetIssueCapsule assetIssueCapsule = createAssetIssue(id, NAME);
+    assetIssueStore.put(assetIssueCapsule.createDbKey(), assetIssueCapsule);
+  }
+
+  private AssetIssueCapsule createAssetIssue(long id, String name) {
```
[the remark this answers is not in the document]
Comment (forfreeday) [reply framework/src/test/java/org/tron/core/db/AssetIssueStoreTest.java:45]: necessary, which is the ID in two different test cases

File: framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java (hunk 2 above)
Comment (forfreeday) [reply framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java:91]: It is possible not to throw this exception

Comment (lurais) [reply framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java:91]: Shall it be throw out but not be caught ?

**Pull request closed without merging**